src/main/webapp/stylesheets/teammates.css

```diff
@@ -96,7 +97,10 @@ h2 {
 /*-----------------------------------------------------------------------------------------*/
 
 #header {
-    background-color: black;
+   /* background-color: black; */
+ /*   background-color: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#1e5799), color-stop(50%,#2989d8), color-stop(51%,#207cca), color-stop(100%,#7db9e8)); */
+  /*  background-color: -moz-linear-gradient(top, #1e5799 0%, #2989d8 50%, #207cca 51%, #7db9e8 100%); */
+    background-color: #408080; 
```

Comment:
These changes to the css do not look like they are used for this issue, so you should remove them.